NuRadioReco/detector/antennapattern.py

```diff
@@ -67,9 +61,9 @@ def interpolate_linear_vectorized(x, x0, x1, y0, y1, interpolation_method='compl
     mask = x0 != x1
     result = np.zeros_like(x, dtype=np.complex)
     denominator = x1 - x0
-    if(interpolation_method == 'complex'):
+    if interpolation_method == 'complex':
```

Comment:
why are the parentheses removed?

Reply:
I thought this was part of pep8, but I checked and it doesn't seem to be explicitly stated. If you prefer, we can get rid of this check.

Reply:
yes I'd remove it. I personally find it more readable with parentheses.